Update Hausa locale information

petyaraykovska opened the pull request:
Request: https://make.wordpress.org/polyglots/2016/04/22/request-for-adding-hausa-locale/

petyaraykovska committed:
Update Hausa locale information

petyaraykovska committed:
Fix typo on line 48

toolstack commented on locales/locales.php:
```diff
@@ -961,15 +961,15 @@ public function __construct() {
+		$hau->lang_code_iso_639_1 = 'ha';
+		$hau->lang_code_iso_639_2 = 'hau';
+		$hau->lang_code_iso_639_3 = 'hau';
+		$hau->slug = 'hau';
```
Any particular reason for the slug change?

petyaraykovska replied:
We're using ISO 639-3 codes for new WordPress locales and where possible adapting old ones as well (https://make.wordpress.org/polyglots/handbook/translating/requesting-a-new-locale/#find-your-language-code). Hausa has existed in GlotPress for a while but it doesn't have a WP locale and didn't have a site. The ISO 639-3 code is applied to the slugs as well as the language code for consistency. The site is already created with the slug `hau`.

toolstack commented on locales/locales.php:
```diff
@@ -961,15 +961,15 @@ public function __construct() {
-		$ha->facebook_locale = 'ha_NG';
+		$hau = new GP_Locale();
+		$hau->english_name = 'Hausa';
+		$hau->native_name = 'Harshen Hausa';
```
The native name should not be in english, this should be in the locale language.

petyaraykovska replied:
Languages are evolving. The locale was originally added in Arabic script, including text direction. Both script and text direction are changed in this version at the request of the person who will maintain the locale. The native name is not in English, it's in boko, which is a latin alphabeth used for Hausa. https://en.wikipedia.org/wiki/Boko_alphabet

petyaraykovska committed:
Adds wp_locale language code

ocean90 commented:
@petyaraykovska Instead of updating the existing locale, let's add `hau` as a new one.

petyaraykovska committed:
Add hau, bring back ha, change ky to kir

As per @ocean90's request, adding a new locale for Hausa in Latin script and bringing back the data for the old rtl locale. As per a GTE request & information on Ethnologue and Wikipedia, also fixing the Kirghiz locale code (former ky, which is the wrong code for the language) and exchanging the locale codes with the ISO 639-3 code.

ocean90 commented on locales/locales.php:
```diff
@@ -962,14 +962,14 @@ public function __construct() {
+ -		$ha->slug = 'ha';
+ -		$ha->text_direction = 'rtl';
+ -		$ha->google_code = 'ha';
+ -		$ha->facebook_locale = 'ha_NG';
```
The `-` shouldn't be there...

ocean90 commented:
Please create one PR per locale.

ocean90 commented:
New PRs: #549 and #550
